modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DDMFolderSelector.js

```diff
@@ -74,16 +74,18 @@ const DDMFolderSelector = ({
 		const bodyContentObject = objectToFormData(
 			itemType === 'folder'
 				? {
-						[`${portletNamespace}sourceRepositoryId`]: sourceRepositoryId,
-						[`${portletNamespace}sourceFolderId`]: sourceFolderId,
-						[`${portletNamespace}destinationParentFolderId`]: destinationParentFolderId,
-						[`${portletNamespace}destinationRepositoryId`]: destinationRepositoryId,
+						[`destinationParentFolderId`]: destinationParentFolderId,
```

Comment:
Just to be sure, did you check that without the `portletNamespace` these attributes are received by the servlet? 
Usually `portletNamespace`  is needed

Reply:
This change was made because on the servlet calls it wasn't needed.
Maybe @adolfopa can confirm. :smile:

Reply:
Servlets don't use namespaces. That's something specific to portlets.